src/components/translation/TranslationDirective.js

```diff
@@ -38,6 +39,10 @@ goog.require('ga_translation_service');
         scope.$on('gaTopicChange', function(event, newTopic) {
           scope.langs = newTopic.langs;
         });
+
+        gaTopic.loadConfig().then(function() {
+          scope.langs = gaTopic.get().langs;
+        });
```

Comment:
if we use `loadConfig`, wee need to register the callback on gaTopicChange here too to avoid unwanted calls.